usr/share/rear/output/default/950_copy_result_files.sh

```diff
@@ -96,11 +96,15 @@ case "$scheme" in
         done
         ;;
     (fish|ftp|ftps|hftp|http|https|sftp)
+        if [[ ! "$OUTPUT_LFTP_OPTIONS" =~ ';'$ ]] ; then
```

Comment:
This condition also matches when $OUTPUT_LFTP_OPTIONS is empty
so that by default OUTPUT_LFTP_OPTIONS becomes a single `;` 
that is added to the `lftp` command.